dbt_meshify/dbt_projects.py

```diff
@@ -156,7 +190,9 @@ def __init__(
         self.dbt = dbt
         self.file_manager = DbtFileManager(read_project_path=path, write_project_path=path)
 
-    def select_resources(self, select: str, exclude: Optional[str] = None, output_key: Optional[str] = None) -> Set[str]:
+    def select_resources(
```

Comment:
aside: would love to also have a) black as part of the requirements and b) some pre-commit stuff setup so that we are all using it consistently!

Reply:
Strong agree!